themes/default/components/src/components/swiper/swiper.tsx

```diff
@@ -68,45 +71,39 @@ export class Swiper {
 
         SwiperJS.use(modules);
 
-        this.containerClass = `swiper-container-${this.swiperHash}`;
-        this.nextBtnClass = `swiper-button-next-${this.swiperHash}`;
-        this.previousBtnClass = `swiper-button-prev-${this.swiperHash}`;
+        this.containerClass = `swiper-container-${this.swiperID}`;
```

Comment:
Just for clarity, since this seems to refer to some sort of randomish unique identifier.

Reply:
👍 Without the ID you get some unexpected behavior with multiple swipers on one page.